### DISCUSSION TITLE
Add a payment gateway using Midtrans or Tripay.

### DISCUSSION THREAD

**Discussion by ardvz** [Paymenter/Paymenter#301] [Ideas]
Maybe it's too much but most people in my country use Midtrans or Tripay and if it was on Paymenter it would probably attract users in my country. Maybe the developer is interested in adding Midtrans or Tripay to this Paymenter and if this is accepted, I would be very grateful.

**Comment by annwarsa** [comment]
i think xendit already have qris

**Comment by CorwinDev** [comment]
If possible please send my sandbox details for either of those gateways via mail(corwin@paymenter.org) or discord (corwindev)
